packages/css/src/components/card/readme.md

```diff
@@ -0,0 +1,3 @@
+# `.dso-card`
+
+Er dient door de implementator doormiddel van script een doorklik gemaakt te worden op de `.dso-card`
```

Comment:
Waar is dit voor? Juist hierom hebben we toch een markup voorschrift met een `<a>` ?